benchmarks/python/test_cross_entropy_loss.py

```diff
@@ -80,3 +83,49 @@ def fwd_call(inp):
         [outputs[0], test_case.grads(), *fwd_inputs, *list(model.parameters())],
         iobytes=test_case.grad_iobytes(),
     )
+
+
+@pytest.mark.parametrize(
+    "executor", ["eager", "torchcompile", "thunder", "thunder-torchcompile"]
+)
+@pytest.mark.parametrize("vocab_size", SyntheticMiniModel.generate_vocab_sizes())
+def test_cross_entropy_mini_benchmark_fwd(benchmark, executor: str, vocab_size: int):
+    if executor == "torchcompile":
+        clear_dynamo_cache()
+
+    # picking a value that doesn't OOM for large vocab sizes
+    batch_size = 4096
+
+    def fwd_call(inp):
+        return SyntheticMiniModel.mini_model(*inp)
+
+    input, target = SyntheticMiniModel.inputs(batch_size, vocab_size)
+
+    kwargs = {}
+    inputs = [input, target]
+    fwd_fn = with_executor(executor, fwd_call, **kwargs)
+    run_benchmark(benchmark, fwd_fn, inputs)
+
+
+@pytest.mark.parametrize(
+    "executor", ["eager", "torchcompile", "thunder", "thunder-torchcompile"]
+)
+@pytest.mark.parametrize("vocab_size", SyntheticMiniModel.generate_vocab_sizes())
+def test_cross_entropy_mini_benchmark_bwd(benchmark, executor: str, vocab_size: int):
+    if executor == "torchcompile":
+        clear_dynamo_cache()
+
+    # picking a value that doesn't OOM for large vocab sizes
+    batch_size = 4096
+
+    def fwd_call(inp):
+        return SyntheticMiniModel.mini_model(*inp)
+
+    input, target = SyntheticMiniModel.inputs(batch_size, vocab_size)
+
+    kwargs = {}
+    inputs = [input, target]
+    fwd_fn = with_executor(executor, fwd_call, **kwargs)
+    outputs = fwd_fn(inputs)
+    grads = SyntheticMiniModel.grads()
+    run_benchmark(benchmark, unary_bwd_torch, [outputs, grads, inputs])
```

Comment:
I think you need to flatten `inputs` here.
Since our io computation doesn't do tree flatten on inputs.

Reply:
Can you explain this a bit - I wasn't quite sure what it meant.